modules/images-other-jenkins-agent-images.adoc

```diff
@@ -11,10 +11,10 @@ The {product-title} Jenkins agent images are available on `quay.io` or
 Jenkins images are available through the Red Hat Registry:
 
 ----
-$ docker pull registry.redhat.io/openshift4/ose-jenkins:<v4.1.4>
-$ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-nodejs:<v4.1.4>
-$ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-maven:<v4.1.4>
-$ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-base:<v4.1.4>
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins:<v4.2.0>
```

Comment:
Is there a reason that registry was changed to a misspelling of regsitry here? (And another instance in a file below)

Reply:
@bergerhoffer Good catch! Not sure what happened there, but all fixed.